### PR TITLE
symfony 4 upgrade

### PR DESCRIPTION
**Pull request by tibineogen**
#SymfonyConHackday2017

### CODE REVIEW

**Comment by craue** [comment]
I prefer to wait for the release of Symfony 4.0. Would you update the PR once that happened?

File: .travis.yml [hunk 1]
```diff
@@ -28,6 +28,8 @@ matrix:
       env: SYMFONY_VERSION='3.3.*'
     - php: 5.6
       env: SYMFONY_VERSION='3.4.*' MIN_STABILITY='dev'
+    - php: 7.1
+      env: SYMFONY_VERSION='4.*' DEPENDENCIES='beta'
   allow_failures:
     - env: SYMFONY_VERSION='3.4.*' MIN_STABILITY='dev'
```
Comment (craue) [inline .travis.yml]: Please make it look like this:

```yaml
    - php: 5.6
      env: SYMFONY_VERSION='3.4.*'
    - php: 7.1
      env: SYMFONY_VERSION='4.0.*'
    - php: 7.1
      env: SYMFONY_VERSION='4.1.*' MIN_STABILITY='dev'
  allow_failures:
    - env: SYMFONY_VERSION='4.1.*' MIN_STABILITY='dev'
```

File: .travis.yml [hunk 2]
```diff
@@ -36,6 +38,8 @@ services:
   - postgresql
 
 install:
+  # suport for symfony 4.0
+  - if [ "$DEPENDENCIES" = "beta" ]; then composer config minimum-stability beta; fi;
```
Comment (craue) [inline .travis.yml]: This change can be removed as `MIN_STABILITY` is used (and handled) already. There's no reason to introduce `DEPENDENCIES`.

File: .travis.yml [hunk 3]
```diff
@@ -27,9 +27,13 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION='3.3.*'
     - php: 5.6
-      env: SYMFONY_VERSION='3.4.*' MIN_STABILITY='dev'
+      env: SYMFONY_VERSION='3.4.*'
+    - php: 7.1
+      env: SYMFONY_VERSION='4.0.*'
+    - php: 7.1
+      env: SYMFONY_VERSION='4.1.*' DEPENDENCIES='dev'
```
Comment (craue) [inline .travis.yml]: `MIN_STABILITY` 😏

**Comment by craue** [comment]
And would you squash all commits into one when done?

**Comment by Tilotiti** [comment]
I would be so greatfull if this pull could be out today. I've a big delivery tomorow and this is the last thing I need to finish.

You are the best <3

**Comment by Tilotiti** [comment]
You are definitively the best ❤️

**Comment by tibineogen** [comment]
Thanks, I hope I did not hold up your project for too much :)

**Comment by craue** [comment]
It's fine, @tibineogen. Thank you for contributing.

...and wish fulfilled, @Tilotiti. :angel: 1.5.0 is out. :sparkles:

**Comment by Tilotiti** [comment]
Just for information, the fixture method doesn't work anymore with SF4 and doctrine/doctrine-fixtures-bundle v3.0.2.

The Option --fixtures disappeared so we are not able to load it anymore.

**Comment by craue** [comment]
True. Any idea on how to load individual fixtures now?

**Comment by Tilotiti** [comment]
I didn't used the fixture method (wich is supposed to be only on development environment). I used a command to integrate the zipcode file.

We maybe could adapt this command, implement it in the bundle, and just add an input option to locate the desired file.

https://gist.github.com/Tilotiti/5b69e938475abc896f902d5ea7d3ff93

**Comment by craue** [comment]
I also thought about adding a command instead, but a fixture is convenient and flexible. So I prefer waiting for a solution within DoctrineFixturesBundle itself, probably by https://github.com/doctrine/DoctrineFixturesBundle/issues/219#issuecomment-352337579 becoming implemented.

**Pull request merged**